src/GitObjectDb/Services/ComputeTreeChanges.cs

```diff
@@ -281,7 +280,6 @@ where UniqueId.TryParse(part, out tempId)
                                                 let id = tempId
                                                 select id);
             var context = new ModelObjectSerializationContext(_container);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove the unused local variable 'context'. ([csharpsquid:S1481](https://sonarcloud.io/organizations/frblondin-github/rules#rule_key=csharpsquid%3AS1481))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=GitObjectDb&pullRequest=40&issues=AWglHPq9Si5GWQS9pJfi&open=AWglHPq9Si5GWQS9pJfi)

[ISSUE_KEY]: AWglHPq9Si5GWQS9pJfi